### ISSUE TITLE
"Cover" Related Items are all missing their textures

### ISSUE THREAD

**Issue by Devilswarchild** [refinedmods/refinedstorage#3580]
### Describe the bug

anything with "Cover" is missing its texture
![Screenshot_2](https://github.com/refinedmods/refinedstorage/assets/27385258/5c844530-6414-45df-a1f2-03c588adaa6a)

### How can we reproduce this bug or crash?

load up my modpack and search "Cover" in the JEi gui
[MineColonies Dimensional Adventure-1.20.1 - 1.9.6 Alpha 2.zip](https://github.com/refinedmods/refinedstorage/files/13196735/MineColonies.Dimensional.Adventure-1.20.1.-.1.9.6.Alpha.2.zip)

### What Minecraft version is this happening on?

Minecraft 1.20.1

### What Forge version is this happening on?

47.2.1

### What Refined Storage version is this happening on?

1.12.3

### Relevant log output

```shell
https://mclo.gs/MrkAGVs - debug.log

latest.tx - https://mclo.gs/pmbDH75
```

**Comment by raoulvdberge** [comment]
Hi, this is most likely a compatibility issue with an other rendering mod. Please try to reproduce a minimal case with just RS installed.

**Comment by embeddedt** [comment]
This is a bug in ModernFix which will be solved in the next release (5.9.1+).

**Comment by raoulvdberge** [comment]
Thank you for letting me know!
